### PR TITLE
Orodiuk create follow entity dao service controller

### PR DESCRIPTION
**Pull request by Shurara**

### CODE REVIEW

File: mentionserver/src/main/java/com.mention/controller/FollowController.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  @PostMapping
+  public void addFollow(@RequestBody Follow follow) {
+    followService.addFollow(follow);
+  }
+
+  @PutMapping
+  public void updateFollow(@RequestBody Follow follow) {
+    followService.updateFollow(follow);
+  }
```
Comment (kKhairulin) [inline mentionserver/src/main/java/com.mention/controller/FollowController.java]: Follow can't be updated. Consider removing it from service and controller

File: mentionserver/src/main/java/com.mention/model/Follow.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          {"profile", "sentMessages", "posts",
+                  "comments", "receivedMessages",
+                  "chats", "favorites", "followers", "followedUsers"})
+  private User followedUser;
+
+ /* @CreatedDate
+  @Temporal(TemporalType.TIMESTAMP)
+  @Column(nullable = false, name = "fw_timestamp", updatable = false)
+  private Date timestamp;
+
```
Comment (kKhairulin) [inline mentionserver/src/main/java/com.mention/model/Follow.java]: Uncomment @CreatedDate

File: mentionserver/src/main/java/com.mention/model/Follow.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  @LastModifiedDate
+  @Temporal(TemporalType.TIMESTAMP)
+  @Column(name = "fw_modify_timestamp")
+  private Date modifyTimestamp;
+*/
```
Comment (kKhairulin) [inline mentionserver/src/main/java/com.mention/model/Follow.java]: Remove @LastModifiedDate

File: mentionserver/src/main/java/com.mention/model/Follow.java (hunk 2 above)
Comment (Shurara) [reply mentionserver/src/main/java/com.mention/model/Follow.java]: corrected

**Pull request merged**